src/frontend/ForumIndex.js

```diff
@@ -11,6 +11,28 @@ export default class ForumIndex extends React.Component {
     }
 
     render() {
-        return (<></>);
+        return (<>
```

Comment:
You should render the array of ThreadLinks that we are storing in the ForumIndex's state, for example:
```
return (<div className='someContainer>
    { this.state.threadLinks }
<div>);